src/network/connection.cpp

```diff
@@ -2102,17 +2109,16 @@ void ConnectionReceiveThread::receive()
 
 	unsigned int loop_count = 0;
 
-	/* first of all read packets from socket */
-	/* check for incoming data available */
-	while( (loop_count < 10) &&
-			(m_connection->m_udpSocket.WaitData(50))) {
+	// First read packets from socket
+	// Check for incoming data available
+	while ((loop_count < 10) && wait(100 * 1000, readset)) {
```

Comment:
It's `100 * 1000` microseconds (us), that's `100` miliseconds (ms) or `0.1` seconds (s).
And it doesn't really make sense for this to be configurable.  You can't really optimize anything by changing it.